pages/folder/components/EditBox.tsx

```diff
@@ -105,9 +105,16 @@ export default function EditBox({
         buttonText="변경하기"
         color="linear-gradient"
         handleClickButton={(value) => {
-          const requestData = { name: value as string };
-          editMutation.mutate(requestData);
+          if (typeof value === "string") {
+            if (value.length < 10) {
+              const requestData = { name: value };
+              editMutation.mutate(requestData);
```

Comment:
이정도는 변수로 할당하지 않고 inline으로 바로 쓰셔도 좋습니다.
`editMutation.mutate({ name: value });` 이런식으로요.
```suggestion
             editMutation.mutate({ name: value });
```